### PR TITLE
remove unused custom easyblock for TAU + clean up custom easyblock for Paraver, only support Paravar >= v4.7

### PR DESCRIPTION
**Pull request by boegel**
This is causing some trouble in https://github.com/easybuilders/easybuild-easyblocks/pull/3534, but we haven't had any easyconfigs that rely on the custom easyblock for TAU in ages, so let's trash it...

We can also remove the dead code that was there to support Paraver < v4.7, oldest Paraver easyconfig we still have in `5.0.x` branch is `Paraver-4.9.2-foss-2021a.eb`...

### CODE REVIEW

**Review by branfosj — DISMISSED** [review]

File: easybuild/easyblocks/p/paraver.py [hunk 1]
```diff
@@ -107,53 +78,18 @@ def configure_step(self):
...
+        self.cfg.update('configopts', '--with-paraver=%s' % self.installdir)
+        self.cfg.update('configopts', '--with-wx-config=%s' % wx_config)
+        # wxPropertyGrid is not required with recent wxWidgets
+        if wxpropgrid:
+            self.cfg.update('configopts', '--with-wxpropgrid=%s' % wxpropgrid)
 
         super(EB_Paraver, self).configure_step()
 
     def build_step(self):
         """Custom build procedure for Paraver: skip 'make' for recent versions."""
```
Comment (branfosj) [inline easybuild/easyblocks/p/paraver.py:91]:
```suggestion
        """No build ('make') required for recent versions."""
```

File: easybuild/easyblocks/p/paraver.py [hunk 2]
```diff
@@ -88,8 +61,6 @@ def configure_step(self):
         wxwidgets = get_software_root('wxWidgets')
         if wxwidgets:
             wx_config = os.path.join(wxwidgets, 'bin', 'wx-config')
-        elif LooseVersion(self.version) >= LooseVersion('4.7'):
-            raise EasyBuildError("wxWidgets is not available as a dependency")
```
Comment (branfosj) [inline easybuild/easyblocks/p/paraver.py:92]: This is still need, just not behind the `elif`?

Comment (boegel) [reply easybuild/easyblocks/p/paraver.py]: fixed in ee1ee1473

**Review by branfosj — DISMISSED** [review]

**Review by branfosj — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _Paraver-4.11.1-foss-2022a.eb_ 
 * **SUCCESS** _Paraver-4.9.2-foss-2021a.eb_ 

Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3569.doduo.os - Linux RHEL 8.8, x86_64, AMD EPYC 7552 48-Core Processor, Python 3.6.8
See https://gist.github.com/boegel/6e247e15bdb8d8f0d2b414648593c04b for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _Paraver-4.11.1-foss-2022a.eb_ 

Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.10, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/4af829bc6a80946c94bf05fc8f845d60 for a full test report.

**Pull request merged**